sharkfuser/build_tools/cmake/CTestMacros.cmake

```diff
@@ -29,6 +43,33 @@ function(_add_sharkfuser_target)
     target_link_options(${_RULE_NAME} PRIVATE -coverage)
   endif()
 
+  # Place executable in the build/bin sub-directory
+  set_target_properties(
+      ${_RULE_NAME} PROPERTIES
+      RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin/${_RULE_BIN_SUBDIR}
+  )
+endfunction()
+
+
+# Creates and executable target + test.
+function(_add_sharkfuser_test)
```

Comment:
As this is mainly adding a ctest target, call this `_add_sharkfuser_ctest_target`?